shiny/observatory/app.R

```diff
@@ -1,6 +1,88 @@
 library(shiny)
+library(tidyverse)
```

Comment:
Do we need all of tidyverse or just dplyr?

Reply:
I use ggplot, stringr, plyr and magrittr which are all in the tidyverse package. Date I will also use lubridate so i find it easier to just use tidyverse